src/CreateEventQueryView/CreateEventQueryView.js

```diff
@@ -1,61 +1,134 @@
 import {RaisedButton} from "material-ui";
 import IconSave from "material-ui/svg-icons/content/save";
 import IconCancel from "material-ui/svg-icons/navigation/cancel";
 import Header from './../Header';
-import { css } from 'aphrodite';
+import {css} from 'aphrodite';
 import EventTypeInformation from "./EventTypeInformation";
 import EventQueryInputArea from "./EventQueryInputArea";
 import config from "../config/config";
 import AppStyles from "../AppStyles";
+import ErrorMessage from '../Utils/ErrorMessage.js';
 
 class CreateEventQueryView extends ConnectionComponent {
-    render() {
-        const allFetches = PromiseState.all([this.props.eventType, this.props.attributes]);
-        const eventType = this.props.eventType.value;
-        const attributes = this.props.attributes.value;
-        const connectionIncomplete = super.render(allFetches);
-        if(connectionIncomplete) {
-            return connectionIncomplete;
-        }
-        return (
-            <div>
-                <Header title={"Create Event Query for " + eventType.Name}/>
-                <div className={AppStyles.elementMarginTop}>
-                    <Container>
-                        <Row>
-                            <Col md={4}>
-                                <EventTypeInformation attributes={attributes}/>
-                            </Col>
-                            <Col md={8}>
-                                <EventQueryInputArea/>
-                            </Col>
-                        </Row>
-                        <div className={css(AppStyles.textAlignCenter)}>
-                            <RaisedButton
-                                label="Abort"
-                                icon={<IconCancel/>}
-                                className={css(AppStyles.marginAllSites)}
-                                secondary={true}
-                            />
-                            <RaisedButton
-                                label="Save"
-                                icon={<IconSave/>}
-                                className={css(AppStyles.marginAllSites)}
-                                primary={true}
-                            />
-                        </div>
-                    </Container>
-                </div>
-            </div>
-        );
-    }
+
+	constructor(props) {
+		super(props);
+		this.state = {
+			query: '',
+			queryErrorMessage: '',
+			queryDescription: '',
+			descriptionErrorMessage: ''
+		};
+		this.handleQueryInput = this.handleQueryInput.bind(this);
+		this.submitQuery = this.submitQuery.bind(this);
+		this.handleDescriptionInput = this.handleDescriptionInput.bind(this);
+	}
+	
+	handleQueryInput(event) {
+		this.setState({
+			query: event.target.value,
+			queryErrorMessage: ''
+		});
+	}
+
+	handleDescriptionInput(event) {
+		this.setState({
+			queryDescription: event.target.value,
+			descriptionErrorMessage: ''
+		});
+	}
+	
+	submitQuery() {
```

Comment:
I do not get the point of your comment.
SubmitQuery calls _createEventQuery_ in our connector (line  127-133), which triggers a request to /eventquery/create